diffus-derive/src/lib.rs

```diff
@@ -253,7 +265,9 @@ pub fn derive_diffus(input: proc_macro::TokenStream) -> proc_macro::TokenStream
                     }
                 }
 
-            })
+            };
+
+            proc_macro::TokenStream::from(result)
```

Comment:
Why not inlined anymore?

Reply:
did it for printf debugging, but thought that it could be a first step of moving out all the `proc_macro::TokenStream::from(...)` so I kept it